integration/tracing/tracing-opentelemetry/pom.xml

```diff
@@ -46,6 +46,10 @@
             <artifactId>jakarta.xml.ws-api</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>jakarta.servlet</groupId>
+            <artifactId>jakarta.servlet-api</artifactId>
```

Comment:
```suggestion
            <optional>true</optional>
```

Reply:
(sorry weird diff, but could you please add optional, we don't want to bundle servlet-api with this module)